test/src/kbasesearchengine/test/main/SearchMethodsTest.java

```diff
@@ -9,16 +9,11 @@
 import java.util.Arrays;
 import java.util.Collections;
 
+import kbasesearchengine.*;
```

Comment:
Please don't use wildcard imports

Reply:
Done. (Was there in the original code).

Reply:
If it was in the original code why did it show up in the diff - not getting this...?